src/status_im2/contexts/quo_preview/switcher/base_card.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im2.contexts.quo-preview.switcher.base-card
+  (:require [quo.react-native :as rn]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [status-im2.common.resources :as resources]))
+
+(def descriptor
+  [{:label "Banner?"
+    :key   :banner?
+    :type  :boolean}
+   (preview/customization-color-option)])
+
+(def banner {:source (resources/get-mock-image :community-banner)})
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:customization-color :sky
+                             :banner?             false})]
+    (fn []
+      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+       [rn/view {:padding-bottom 150}
+        [preview/customizer state descriptor]
+        [rn/view
+         {:style {:padding-vertical 60
+                  :align-items      :center}}
+         [quo/switcher-base-card
+          (merge @state
+                 {:title    "jonathan.eth"
+                  :subtitle "Message"
+                  :banner   (when (:banner? @state) banner)})]]]])))
+
+(defn preview-base-card
+  []
+  [rn/view
+   {:background-color (colors/theme-colors colors/white colors/neutral-90)
```

Comment:
Quick one, I didn't spot before. The styles should be inside the `:style` prop. The file src/status_im2/contexts/quo_preview/switcher/group_messaging_card.cljs has 2 instances of this problem too.